[Bug][CPU Backend] [Arm]: FAILED tests/kernels/moe/test_moe.py::test_cpu_fused_moe_basic[silu-False-dtype0-2-8-128-128-1]

### Your current environment

<details>
<summary>The output of <code>python collect_env.py</code></summary>

```text
==============================
        System Info
==============================
OS                           : Ubuntu 22.04.5 LTS (aarch64)
GCC version                  : (Ubuntu 12.3.0-1ubuntu1~22.04.2) 12.3.0
Clang version                : Could not collect
CMake version                : Could not collect
Libc version                 : glibc-2.35

==============================
       PyTorch Info
==============================
PyTorch version              : 2.9.1+cpu
Is debug build               : False
CUDA used to build PyTorch   : None
ROCM used to build PyTorch   : N/A

==============================
      Python Environment
==============================
Python version               : 3.12.12 (main, Jan 14 2026, 19:35:10) [Clang 21.1.4 ] (64-bit runtime)
Python platform              : Linux-6.14.0-1018-aws-aarch64-with-glibc2.35

==============================
       CUDA / GPU Info
==============================
Is CUDA available            : False
CUDA runtime version         : No CUDA
CUDA_MODULE_LOADING set to   : N/A
GPU models and configuration : No CUDA
Nvidia driver version        : No CUDA
cuDNN version                : No CUDA
HIP runtime version          : N/A
MIOpen runtime version       : N/A
Is XNNPACK available         : True

==============================
          CPU Info
==============================
Architecture:                            aarch64
CPU op-mode(s):                          64-bit
Byte Order:                              Little Endian
CPU(s):                                  64
On-line CPU(s) list:                     0-63
Vendor ID:                               ARM
Model name:                              Neoverse-V2
Model:                                   1
Thread(s) per core:                      1
Core(s) per cluster:                     64
Socket(s):                               -
Cluster(s):                              1
Stepping:                                r0p1
BogoMIPS:                                2000.00
Flags:                                   fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 asimddp sha512 sve asimdfhm dit uscat ilrcpc flagm sb paca pacg dcpodp sve2 sveaes svepmull svebitperm svesha3 flagm2 frint svei8mm svebf16 i8mm bf16 dgh rng bti
L1d cache:                               4 MiB (64 instances)
L1i cache:                               4 MiB (64 instances)
L2 cache:                                128 MiB (64 instances)
L3 cache:                                36 MiB (1 instance)
NUMA node(s):                            1
NUMA node0 CPU(s):                       0-63
Vulnerability Gather data sampling:      Not affected
Vulnerability Ghostwrite:                Not affected
Vulnerability Indirect target selection: Not affected
Vulnerability Itlb multihit:             Not affected
Vulnerability L1tf:                      Not affected
Vulnerability Mds:                       Not affected
Vulnerability Meltdown:                  Not affected
Vulnerability Mmio stale data:           Not affected
Vulnerability Reg file data sampling:    Not affected
Vulnerability Retbleed:                  Not affected
Vulnerability Spec rstack overflow:      Not affected
Vulnerability Spec store bypass:         Mitigation; Speculative Store Bypass disabled via prctl
Vulnerability Spectre v1:                Mitigation; __user pointer sanitization
Vulnerability Spectre v2:                Mitigation; CSV2, BHB
Vulnerability Srbds:                     Not affected
Vulnerability Tsa:                       Not affected
Vulnerability Tsx async abort:           Not affected
Vulnerability Vmscape:                   Not affected

==============================
Versions of relevant libraries
==============================
[pip3] mypy_extensions==1.1.0
[pip3] numpy==1.26.4
[pip3] open_clip_torch==2.32.0
[pip3] pyzmq==27.1.0
[pip3] sentence-transformers==5.2.0
[pip3] torch==2.9.1+cpu
[pip3] torchaudio==2.9.1
[pip3] torchvision==0.24.1
[pip3] transformers==4.57.3
[pip3] transformers-stream-generator==0.0.5
[pip3] tritonclient==2.51.0
[pip3] vector-quantize-pytorch==1.27.19
[conda] Could not collect

==============================
         vLLM Info
==============================
ROCM Version                 : Could not collect
vLLM Version                 : 0.14.0rc2.dev171+gbb9172030 (git sha: bb9172030)
vLLM Build Flags:
  CUDA Archs: Not Set; ROCm: Disabled
GPU Topology:
  Could not collect

==============================
     Environment Variables
==============================
VLLM_CPU_KVCACHE_SPACE=16
PYTORCH_NVML_BASED_CUDA_CHECK=1
TORCHINDUCTOR_COMPILE_THREADS=1```

</details>


### 🐛 Describe the bug

```bash
.buildkite/scripts/hardware_ci/run-cpu-test-arm.sh 

```

```
============================= test session starts ==============================
platform linux -- Python 3.12.12, pytest-9.0.2, pluggy-1.6.0 -- /opt/venv/bin/python3
cachedir: .pytest_cache
hypothesis profile 'default'
rootdir: /workspace
plugins: shard-0.1.2, mock-3.15.1, cov-7.0.0, buildkite-test-collector-0.1.9, asyncio-1.3.0, typeguard-4.4.4, schemathesis-4.9.2, timeout-2.4.0, rerunfailures-16.1, forked-1.6.0, hypothesis-6.150.2, anyio-4.12.1
asyncio: mode=Mode.STRICT, debug=False, asyncio_default_fixture_loop_scope=None, asyncio_default_test_loop_scope=function
collecting ... INFO 01-21 16:36:29 [scheduler.py:229] Chunked prefill is enabled with max_num_batched_tokens=2048.
INFO 01-21 16:36:29 [vllm.py:618] Asynchronous scheduling is enabled.
INFO 01-21 16:36:29 [vllm.py:625] Disabling NCCL for DP synchronization when using async scheduling.
WARNING 01-21 16:36:29 [interface.py:598] Current platform cpu does not have '__test__' attribute.
WARNING 01-21 16:36:29 [interface.py:598] Current platform cpu does not have '__bases__' attribute.
WARNING 01-21 16:36:29 [interface.py:598] Current platform cpu does not have '__test__' attribute.
WARNING 01-21 16:36:29 [interface.py:598] Current platform cpu does not have '_schemathesis_metadata' attribute.
collected 3551 items / 3543 deselected / 8 selected
Running 8 items in this shard: tests/kernels/moe/test_moe.py::test_cpu_fused_moe_basic[silu-False-dtype0-2-8-128-128-1], tests/kernels/moe/test_moe.py::test_cpu_fused_moe_basic[silu-False-dtype0-2-8-128-128-33], tests/kernels/moe/test_moe.py::test_cpu_fused_moe_basic[silu-False-dtype1-2-8-128-128-1], tests/kernels/moe/test_moe.py::test_cpu_fused_moe_basic[silu-False-dtype1-2-8-128-128-33], tests/kernels/moe/test_moe.py::test_cpu_fused_moe_basic[silu-True-dtype0-2-8-128-128-1], tests/kernels/moe/test_moe.py::test_cpu_fused_moe_basic[silu-True-dtype0-2-8-128-128-33], tests/kernels/moe/test_moe.py::test_cpu_fused_moe_basic[silu-True-dtype1-2-8-128-128-1], tests/kernels/moe/test_moe.py::test_cpu_fused_moe_basic[silu-True-dtype1-2-8-128-128-33]

tests/kernels/moe/test_moe.py::test_cpu_fused_moe_basic[silu-False-dtype0-2-8-128-128-1] FAILEDWARNING 01-21 16:36:30 [interface.py:598] Current platform cpu does not have 'empty_cache' attribute.


=================================== FAILURES ===================================
__________ test_cpu_fused_moe_basic[silu-False-dtype0-2-8-128-128-1] ___________

m = 1, n = 128, k = 128, e = 8, topk = 2, dtype = torch.float32
with_bias = False, activation = 'silu'

    @pytest.mark.parametrize("m", [1, 33])
    @pytest.mark.parametrize("n,k", [(128, 128)])
    @pytest.mark.parametrize("e", [8])
    @pytest.mark.parametrize("topk", [2])
    @pytest.mark.parametrize("dtype", [torch.float32, torch.bfloat16])
    @pytest.mark.parametrize("with_bias", [False, True])
    @pytest.mark.parametrize("activation", ["silu"])
    @pytest.mark.skipif(not current_platform.is_cpu(), reason="CPU only test")
    def test_cpu_fused_moe_basic(m, n, k, e, topk, dtype, with_bias, activation):
        from vllm.model_executor.layers.fused_moe.cpu_fused_moe import CPUFusedMOE
    
        device = "cpu"
        torch.manual_seed(7)
    
        a = torch.randn((m, k), device=device, dtype=dtype) / 10
        w13 = torch.randn((e, 2 * n, k), device=device, dtype=dtype) / 10
        w2 = torch.randn((e, k, n), device=device, dtype=dtype) / 10
        router_logits = torch.randn((m, e), device=device, dtype=dtype)
    
        b1 = b2 = None
        if with_bias:
            b1 = torch.randn((e, 2 * n), device=device, dtype=dtype) / 10
            b2 = torch.randn((e, k), device=device, dtype=dtype) / 10
    
        ref = (
            torch_moe(a, w13, w2, router_logits, topk, b1, b2)
            if with_bias
>           else torch_moe(a, w13, w2, router_logits, topk)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        )

tests/kernels/moe/test_moe.py:1340: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/kernels/utils.py:980: in torch_moe
    return torch_experts(
tests/kernels/utils.py:895: in torch_experts
    tmp2 = act()(tmp1)
           ^^^^^
/opt/venv/lib/python3.12/site-packages/vllm/model_executor/layers/activation.py:79: in __init__
    super().__init__()
/opt/venv/lib/python3.12/site-packages/vllm/model_executor/custom_op.py:44: in __init__
    self._forward_method = self.dispatch_forward()
                           ^^^^^^^^^^^^^^^^^^^^^^^
/opt/venv/lib/python3.12/site-packages/vllm/model_executor/custom_op.py:88: in dispatch_forward
    compilation_config = get_cached_compilation_config()
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/opt/venv/lib/python3.12/site-packages/vllm/config/vllm.py:1439: in get_cached_compilation_config
    return get_current_vllm_config().compilation_config
           ^^^^^^^^^^^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    def get_current_vllm_config() -> VllmConfig:
        if _current_vllm_config is None:
>           raise AssertionError(
                "Current vLLM config is not set. This typically means "
                "get_current_vllm_config() was called outside of a "
                "set_current_vllm_config() context, or a CustomOp was instantiated "
                "at module import time or model forward time when config is not set. "
                "For tests that directly test custom ops/modules, use the "
                "'default_vllm_config' pytest fixture from tests/conftest.py."
            )
E           AssertionError: Current vLLM config is not set. This typically means get_current_vllm_config() was called outside of a set_current_vllm_config() context, or a CustomOp was instantiated at module import time or model forward time when config is not set. For tests that directly test custom ops/modules, use the 'default_vllm_config' pytest fixture from tests/conftest.py.

/opt/venv/lib/python3.12/site-packages/vllm/config/vllm.py:1444: AssertionError
=============================== warnings summary ===============================
<frozen importlib._bootstrap>:488
  <frozen importlib._bootstrap>:488: DeprecationWarning: builtin type SwigPyPacked has no __module__ attribute

<frozen importlib._bootstrap>:488
  <frozen importlib._bootstrap>:488: DeprecationWarning: builtin type SwigPyObject has no __module__ attribute

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/kernels/moe/test_moe.py::test_cpu_fused_moe_basic[silu-False-dtype0-2-8-128-128-1]
!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!
================ 1 failed, 3543 deselected, 2 warnings in 1.82s ================
```

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

--- a/tests/kernels/moe/test_moe.py
+++ b/tests/kernels/moe/test_moe.py
@@ -1310,6 +1310,7 @@ def test_moe_sum(m: int, topk: int, k: int, dtype: torch.dtype):
     opcheck(torch.ops._moe_C.moe_sum, (input, actual))
 
 
+@pytest.mark.usefixtures("default_vllm_config")
 @pytest.mark.parametrize("m", [1, 33])
 @pytest.mark.parametrize("n,k", [(128, 128)])
 @pytest.mark.parametrize("e", [8])

